client/src/app.test.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
```

Comment:
looks like the test was removed but a new one was not added?

Reply:
this was a duplicate test file. For some reason, it was inside my client/src folder when I already had one in my spec folder. So i deleted this file